twingate/internal/model/remote-network.go

```diff
@@ -1,8 +1,19 @@
 package model
 
+const (
+	LocationAWS         = "AWS"
+	LocationAzure       = "AZURE"
+	LocationGoogleCloud = "GOOGLE_CLOUD"
+	LocationOnPremise   = "ON_PREMISE"
+	LocationOther       = "OTHER"
+)
+
+var Locations = []string{LocationAWS, LocationAzure, LocationGoogleCloud, LocationOnPremise, LocationOther} //nolint
```

Comment:
What happens if we add a new Enum value on server side?   will it break TF?

Reply:
new enum value won’t break it
* datasource accepts any value as they fetched from backend side (so we don’t need to validate it)
* on resource side - user should provide this value and terraform validates according to our list of possible locations, so when new enum added on server side - terraform just won’t allow to use this new value until we update list of allowed location options